docs/ML-Agents-Overview.md

```diff
@@ -684,6 +684,23 @@ three network architectures:
 The choice of the architecture depends on the visual complexity of the scene and
 the available computational resources.
 
+### Learning from Variable Length Observations using Attention
+
+Using the ML-Agents Toolkit, it is possible to have agents learn from a
```

Comment:
I think this section should start with an example of where you may have different inputs to help a non-RL expert understand when and why they want to use it. (See subsection below for instance.)